src/quad_expr.jl

```diff
@@ -232,7 +232,7 @@ function Base.copy(q::GenericQuadExpr, new_model::Model)
                     copy(q.qcoeffs), copy(q.aff, new_model))
 end
 
-# Requires that value(::GenericAffExpr, foo) is defined.
+# Requires that value(::GenericAffExpr, foo) and foo(::VarType) is defined.
```

Comment:
Why is "`value(::GenericAffExpr, foo)` is defined" in the requirements ? It is defined and it will work if `foo(::VarType)` is defined

Reply:
Oh, that's what you meant. Good point.